iac-extensions/spring-config/src/test/java/org/sonar/iac/springconfig/plugin/visitors/SpringConfigHighlightingVisitorTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.springconfig.plugin.visitors;
+
+import java.nio.file.Path;
+import org.junit.jupiter.api.Test;
+import org.sonar.iac.common.testing.AbstractHighlightingTest;
+import org.sonar.iac.common.yaml.YamlLanguage;
+import org.sonar.iac.springconfig.parser.SpringConfigParser;
+import org.sonar.iac.springconfig.plugin.SpringConfigExtension;
+
+import static org.sonar.api.batch.sensor.highlighting.TypeOfText.COMMENT;
+import static org.sonar.api.batch.sensor.highlighting.TypeOfText.KEYWORD;
+import static org.sonar.api.batch.sensor.highlighting.TypeOfText.STRING;
+
+class SpringConfigHighlightingVisitorTest extends AbstractHighlightingTest {
+  protected SpringConfigHighlightingVisitorTest() {
+    super(new SpringConfigHighlightingVisitor(), new SpringConfigParser());
+  }
+
+  @Test
+  void shouldHighlightProperties() {
+    highlight(
+      // language=properties
```

Comment:
Why such comment?

Reply:
It enables syntax highlighting for the string content in IDEA. Not sure if it is useful enough to keep, but it can help to double-check that an example has valid syntax.